great_expectations/data_context/types/base.py

```diff
@@ -2449,7 +2326,6 @@ def __init__(  # noqa: C901, PLR0912, PLR0913, PLR0915
         validation_operators=None,
         stores: Optional[Dict] = None,
         data_docs_sites: Optional[Dict] = None,
-        notebooks: Optional[NotebookConfig] = None,
```

Comment:
Should we allow misc fields for backwards compatibility?

Reply:
Maybe just allow `notebooks`, do nothing with it, and raise a `DeprecationWarning`? Technically breaking changes are allowed, but it does feel bad to break existing yaml configs.